integration-testing/test/cl_node/casperlabs_node.py

```diff
@@ -16,6 +16,9 @@ def __init__(self, config):
         self.node = DockerNode(config, socket_volume=self.socket_volume)
         self.name = f'cl_node-{self.config.number}'
 
+    def logs(self):
```

Comment:
```suggestion
    @property
    def logs(self):
```

So you can type `node.logs` not `node.logs()`, if you like.

Reply:
OK, added this.